src/components/ModalExport.tsx

```diff
@@ -86,6 +86,15 @@ class ModalExportInternal extends React.Component<ModalExportInternalProps> {
   async saveStyle() {
     const tokenStyle = this.tokenizedStyle();
 
+    // it is not guaranteed that the File System Access API is available on all
+    // browsers. If the function is not available, a fallback behavior is used.
+    if (typeof window.showSaveFilePicker !== "function") {
```

Comment:
Store this in a variable to be used layer for readability?

Reply:
I created an variable `showSaveFilePickerAvailable` for it. 👍